### ISSUE TITLE
windows specific sbt setup

### ISSUE THREAD

**Issue by fommil** [ensime/ensime.github.io#40]
note to self, this is needed to use `sbt-mode`

```
(setenv "SBT_OPTS" "-Djline.terminal=jline.UnsupportedTerminal")
```

and this is needed to compile `ensime-server`

```
(setenv "JAVA_OPTS" "-XX:MaxPermSize=256m -Xss2m -Xmx2g")
(setenv "JDK_HOME" "C:/Program Files/Java/jdk1.6.0_45")
(setenv "PATH" (concat (getenv "JDK_HOME") "/bin;" (getenv "PATH")))
```

(can probably be smarter about expanding environment variables)

**Comment by yilinwei** [comment]
I'm wondering whether the latter block needs to be added into the docs since `JDK_HOME` is almost always set by developers anyway.

**Comment by fommil** [comment]
this second block is just for hacking on `ensime-server` itself, it's not needed to use ensime.

**Comment by fommil** [comment]
fixed by sbt 0.13.13
